src/main/resources/xjc/endnote/endnote.xsd

```diff
@@ -668,7 +668,7 @@
         </xs:complexType>
     </xs:element>
     <xs:element name="url">
-        <xs:complexType>
+        <xs:complexType mixed="true">
```

Comment:
Ah cool, did not know about this!

Reply:
Yah, it's funny. I just knew that when I was looking up some relevant docs yesterday.

Reply:
Does anyone know whether there is a globally updated "endnote.xsd"?

I am of the community believing that schema schecking is a good thing (json schema, xml schema, ...). (Side note: some say that even the schema should be sent along with the message: https://microservice-api-patterns.org/patterns/structure/elementStereotypes/MetadataElement#)

If a third party releases an schema for their data meaning that data produced by their software can be read in case it conforms to that schema, I would not try to change that schema.

Google lead me to https://forums.zotero.org/discussion/26493/endnote-xml-export-import - however, this his very outdated. Maybe, endnote.xsd is not maintained anymore and we can go ahead to create a fork?

I just wanted to double check.

Reply:
Do you mean a globally standardized version of "endnote.xsd"? 

I do not know much about it yet. But sending messages with its corresponding schema is indeed reasonable in order to parse the info correctedly.

Reply:
Endnote only has a dtd not an xsd https://support.clarivate.com/Endnote/s/article/EndNote-XML-Document-Type-Definition?language=en_US
Seems like the xsd was somehow generated from an xml file. The xsd has been in JabRef since ages